pkg/cloud/azure/actuators/cluster/reconciler.go

```diff
@@ -35,15 +36,15 @@ import (
 // Reconciler are list of services required by cluster actuator, easy to create a fake
 type Reconciler struct {
 	scope            *actuators.Scope
-	certificatesSvc  azure.Service
-	groupsSvc        azure.Service
-	vnetSvc          azure.Service
-	securityGroupSvc azure.Service
-	routeTableSvc    azure.Service
-	subnetsSvc       azure.Service
-	internalLBSvc    azure.Service
-	publicIPSvc      azure.Service
-	publicLBSvc      azure.Service
+	certificatesSvc  services.Service
```

Comment:
please change it back to azure.Service, since services.Service has a stutter.

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/138

Reply:
Reverted that change.